src/main/server/utils.ts

```diff
@@ -4,7 +4,10 @@ export const handleResponse = ({
   response,
   statusCode = 200,
   header = {
-    'Content-Type': 'application/json'
+    'Content-Type': 'application/json',
+    'access-control-allow-headers': '*',
+    'access-control-allow-methods': 'POST, GET, OPTIONS',
+    'access-control-allow-origin': '*'
```

Comment:
'*' will not work in some cases.

Use the referer or host of the request will be better, so called `dynamic cors`.

However, '*' can face many cases so I will merge this PR. We will upgrade it when needed.